.gitignore

```
@@ -33,4 +33,7 @@ yarn-error.log*
 # vercel
 .vercel
 
-.env
\ No newline at end of file
+.env
+
+# vscode
+/.vscode
```

Think we may want to add extension recommendations later on (once we start adding ESLint etc. in all our projects); so it's best to leave this file as is for now.

```suggestion
.env

```

---

fixed as suggested